meta-ros-common/conf/distro/include/ros-common.inc

```diff
@@ -1,6 +1,6 @@
 # OE DISTRO configuration settings common to both ROS 1 and ROS 2 distros.
 #
-# Copyright (c) 2019 LG Electronics, Inc.
```

Comment:
Commit message: I've been using the convention of have a space between ROS and [12].

Reply:
in 'ROS[12]_DISTRO' when I want to express ROS1_DISTRO or ROS2_DISTRO variable?

Reply:
No, just in the commit message, eg, 

> and enable it by default for all ROS2 distributions as well as ROS1 noetic

-> ROS 2 distributions as well as ROS 1 noetic